### PR TITLE
use babel-preset-latest-minimal as default babel preset

### PR DESCRIPTION
**Pull request by gasolin**

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -36,6 +36,11 @@ const config = merge(preset, {
 
 const babelLoader = config.module.loaders.find(l => l.loader.includes('babel'));
 
+// overwrite neutrino-preset-base babel preset
+babelLoader.query.presets = [
+  require.resolve('babel-preset-latest-minimal'));
```
Comment (eliperelman) [inline src/index.js]: There's an extra parenthesis here. Remove the semicolon at the end.

**Comment by gasolin** [comment]
thanks for catching it! updated

File: src/index.js [hunk 2]
```diff
@@ -36,6 +36,11 @@ const config = merge(preset, {
 
 const babelLoader = config.module.loaders.find(l => l.loader.includes('babel'));
 
+// overwrite neutrino-preset-base babel preset
+babelLoader.query.presets = [
+  require.resolve('babel-preset-latest-minimal'))
```
Comment (eliperelman) [inline src/index.js]: There's still one extra closing parenthesis here.

**Comment by gasolin** [comment]
as discussed in https://github.com/mozilla-neutrino/neutrino-preset-base/pull/8
let's use `babel-preset-env` instead

**Pull request closed without merging**